.test-infra/jenkins/job_LoadTests_GBK_Java_Smoke.groovy

```diff
@@ -79,7 +79,23 @@ def smokeTestConfigurations = { datasetName -> [
                         fanout           : 10,
                         iterations       : 1,
                 ]
+        ],
```

Comment:
I'm not really sure if "smoke" tests are used by anyone tbh. If not, I wouldn't bother adding new scenarios here and propose to create an issue to remove smoke tests (unused code, unused jobs). If however they are used then pls ignore this comment :)

@mwalenia could you confirm what is the case here?

Reply:
ok waiting for the confirmation that it is useless

Reply:
Jenkins logs show that nobody uses the smoke tests, I don't think it's worth the hassle to add more of them

Reply:
So I suggest to remove this part @echauchot. :)

and the issue: https://issues.apache.org/jira/browse/BEAM-9559

Reply:
ok, I reverted the cahnge that added the smoke tests. I did not touch to other runners smoke tests, just in case.